src/test_all.py

```diff
@@ -18,63 +18,62 @@
 
 with open("tasks/README.md", 'r') as readmef:
     task_readme_content = " ".join(readmef.readlines())
-
 files = [f for f in listdir(tasks_path) if isfile(join(tasks_path, f))]
+files.sort()
+lastindex=int(files[-1].split('_')[0].split('task')[1])
 for file in files:
```

Comment:
Since the files are sorted according to the index id, I think it's enough to do `for file in files[70:]` and drop all the `lastindex`-relatd code.

Reply:
I guess you meant files[lastindex-70:] as files[70:] can be huge once the task count reaches 500 or 1000. The code files[lastindex-70:] ensures that only 70 tasks are getting evaluated and this way testing will take a couple of minutes only.